src/backend/distributed/sql/udfs/drop_old_time_partitions/latest.sql

```diff
@@ -0,0 +1,38 @@
+-- Heavily inspired by the procedure alter_old_partitions_set_access_method
```

Comment:
I seem to recall we have some style-checking tools that are opinionated about the type of comments we allow in upgrade scripts

@hanefi would know best

Reply:
Please do not use C-style comments in SQL downgrade/upgrade files.

`--` is fine, `/* */` and `//` are not. As we use c preprocessor, they strip those comments and we may end up with errors during packaging.

See https://github.com/citusdata/citus/tree/master/ci#disallow_c_comments_in_migrationssh for more info.

Reply:
I opened an issue to track an improvement to the CI scripts https://github.com/citusdata/citus/issues/5240 . That script should have captured the c-style errors and fail.